Watchman.Discord/Areas/ParserTime.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Devscord.DiscordFramework.Commons.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Watchman.Common.Models;
+
+namespace Watchman.Discord.Areas
+{
+    public class ParserTime
```

Comment:
ta klasa nie ma sensu, bo dokładnie to samo robimy w frameworku

Reply:
to nie wiem... Mam zrobić w taki sposób, że DiscordRequestExtensions będzie używał ParserTime, aby unikać duplikacji kodu?

Reply:
zobacz na BotCommandsPropertyConversionService

Reply:
i co to ma do tego? nie rozumiem

Reply:
że to już w tamtym miejscu powinno być załatwiane

Reply:
po co chcesz robić cokolwiek związanego z DiscordRequestExtensions?
przecież to zadanie polegało na zakończeniu używania DiscordRequest, bo mamy IBotCommand